packages/react-conformance-griffel/package.json

```diff
@@ -30,7 +30,7 @@
     "typescript": "^4.3.0"
   },
   "dependencies": {
-    "@fluentui/react-conformance": "^0.11.0",
+    "@fluentui/react-conformance": "*",
```

Comment:
this is not a very good idea. can you revert to proper version as this is a direct dependency